### PR TITLE
Add an option to commit updated recipe changes automatically

### PR DESCRIPTION
**Pull request by 1ed**

### CODE REVIEW

**Comment by fabpot** [comment]
I don't think we want to commit changes automatically. A human should always look at the proposed changes and validate them. I'm :-1: for this new option.

**Comment by 1ed** [comment]
I thought about that. That is why it is optional. Most of the time the changes are good without any further actions and this speeds up my workflow a lot. I like to include the changelog in the commit message and doing it manually is really slow. The commits still can be reviewed and amended if needed. In my experience if I need to change something it will conflict anyway.

**Comment by fabpot** [comment]
I will let @symfony/mergers chime in here.

**Comment by 1ed** [comment]
Thanks! A commit message will look something like this:

```
Update recipe for doctrine/doctrine-bundle

  * Remove default database URL as it already defined in .env (PR 588 - https://github.com/symfony/recipes/pull/588)
  * [doctrine-bundle] Update URL to use HTTPS and avoid redirection (PR 628 - https://github.com/symfony/recipes/pull/628)
  * Add a DSN example for psql (PR 655 - https://github.com/symfony/recipes/pull/655)
  * [doctrine/doctrine-bundle] dont duplicate the db settings (PR 674 - https://github.com/symfony/recipes/pull/674)
  * Add recipe for DoctrineBundle 1.12 and newer (PR 682 - https://github.com/symfony/recipes/pull/682)
  * [DoctrineBundle] Add charset option to postgres example (PR 691 - https://github.com/symfony/recipes/pull/691)
  * Remove charset config for doctrine-bundle v2 (PR 693 - https://github.com/symfony/recipes/pull/693)
  * Fix doctrine-bundle recipe for prod caching pools (PR 695 - https://github.com/symfony/recipes/pull/695)
  * Update doctrine.yaml (PR 697 - https://github.com/symfony/recipes/pull/697)
  * enable query profiling for debug (PR 704 - https://github.com/symfony/recipes/pull/704)
  * [doctrine-bundle] Docker support (PR 113 - https://github.com/symfony/recipes/pull/113)
  * Fix docker-compose database service name (PR 843 - https://github.com/symfony/recipes/pull/843)
  * listing mysql, pgsql and sqlite DATABASE_URL examples in .env (PR 845 - https://github.com/symfony/recipes/pull/845)
  * Remove extra quotes (PR 852 - https://github.com/symfony/recipes/pull/852)
  * Test (PR 966 - https://github.com/symfony/recipes/pull/966)
  * [doctrine/doctrine-bundle/2.0] Update post-install.txt for postgresql (PR 893 - https://github.com/symfony/recipes/pull/893)
  * add support for override_url (PR 900 - https://github.com/symfony/recipes/pull/900)
  * [doctrine-bundle] dockerfile: use && instead of ; to chain sh commands (PR 903 - https://github.com/symfony/recipes/pull/903)
  * Wrong comment in doctrine/doctrine-bundle 2.3 recipe (PR 933 - https://github.com/symfony/recipes/pull/933)
  * [doctrine/doctrine-bundle] Use dbname_suffix in tests (PR 939 - https://github.com/symfony/recipes/pull/939)
  * [doctrine-bundle] add missing folders in 2.4 (PR 945 - https://github.com/symfony/recipes/pull/945)
  * Update note about server version (PR 947 - https://github.com/symfony/recipes/pull/947)
  * Remove all symlinks (PR 895 - https://github.com/symfony/recipes/pull/895)
  * Remove the header from post-install files (handled by Flex soon) (PR 991 - https://github.com/symfony/recipes/pull/991)
  * [DoctrineBundle] Removed deprecated config (PR 996 - https://github.com/symfony/recipes/pull/996)
  * Rely on defaults for doctrine mapping types (PR 971 - https://github.com/symfony/recipes/pull/971)
  * DATABASE_URL configured like postgresql config (PR 1013 - https://github.com/symfony/recipes/pull/1013)
  * Consolidating env-specific files into main config file for simplicity (PR 1044 - https://github.com/symfony/recipes/pull/1044)
  * Specify charset when using MySQL (PR 1059 - https://github.com/symfony/recipes/pull/1059)
  * Update spacing of doctrine yml files (PR 1018 - https://github.com/symfony/recipes/pull/1018)
  * [doctrine-bundle] Upgrade to Postgres 14 and MySQL 8 (PR 1115 - https://github.com/symfony/recipes/pull/1115)
  * Put doctrine proxies in build dir for production (PR 1143 - https://github.com/symfony/recipes/pull/1143)
  * Use PostgreSQL 14 everywhere (PR 1151 - https://github.com/symfony/recipes/pull/1151)
  * Turn on "doctrine.orm.enable_lazy_ghost_objects" by default (PR 1140 - https://github.com/symfony/recipes/pull/1140)
  * Use PostgreSQL 15 by default (PR 1166 - https://github.com/symfony/recipes/pull/1166)
  * [doctrine-bundle] make Docker volume name consistent (PR 1116 - https://github.com/symfony/recipes/pull/1116)
  * fix(doctrine): adding quote to prevent word splitting in Dockerfile (PR 1180 - https://github.com/symfony/recipes/pull/1180)
```

File: src/Command/UpdateRecipesCommand.php [hunk 1]
```diff
@@ -190,17 +192,27 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 '  Run <comment>git status</comment> to see them.',
                 '  After resolving, commit your changes like normal.',
             ]);
+
+            if (false !== $input->getOption('auto-commit')) {
+                $io->write([
+                    '  For a prepered commit message run <comment>git commit -F .git/SYMFONY_FLEX_COMMIT_MSG</comment>.',
```
Comment (OskarStark) [inline src/Command/UpdateRecipesCommand.php]:
```suggestion
                    '  For a prepared commit message run <comment>git commit -F .git/SYMFONY_FLEX_COMMIT_MSG</comment>.',
```

**Comment by chalasr** [comment]
I also don't think we need further automation on this part.

**Pull request closed without merging**